lib/core/src/Cardano/Wallet/TokenMetadata.hs

```diff
@@ -383,12 +383,12 @@ data TokenMetadataLog
 
 instance HasSeverityAnnotation TokenMetadataLog where
     getSeverityAnnotation = \case
-        MsgNotConfigured -> Notice
+        MsgNotConfigured -> Info
```

Comment:
This _would_ get drowned in the other Info logs. I wonder if there better ways of reminding the user that SMASH isn't configured...

- Print Notice log when listing pools (but maybe not)
- Provide some kind of wallet state in the HTTP headers of the response

Reply:
Ah, this is the asset metadata. Anyway, I'll re-raise this to Notice.

Reply:
This is actually a log message for Token Metadata.
It will be logged exactly once at startup.
So I think Notice is the correct level, right?